packages/components/src/internal/components/editable/Cell.tsx

```diff
@@ -113,6 +117,10 @@ export class Cell extends React.PureComponent<CellProps, State> {
         return col.isPublicLookup() || this.isDateTimeField || !!col.validValues;
     }
 
+    get isMultiline(): boolean {
+        return this.props.col.inputType === 'textarea';
```

Comment:
minor: the `col` here is a QueryColumn right? if so, this`isMultiline` might make sense on that instead since there are other similar helpers on QueryColumn already: isFileInput, isDateOnlyColumn, etc.

Reply:
I'm going to leave as-is since the `QueryColumn` doesn't really care about multi-line. It already provides enough information to be based off of and `inputType` is used in a number of places in a variety of ways.